lib/mayaUsd/resources/ae/usdschemabase/material_custom_control.py

```diff
@@ -0,0 +1,202 @@
+        '''
+        React to change of the strength drop-down choice by updating the direct
+        material binding relationship strength.
+        '''
+        if value not in MaterialCustomControl.strengthTokens:
+            return
+
+        matAPI = UsdShade.MaterialBindingAPI(prim)
+        directBinding = matAPI.GetDirectBinding()
+        directRel = directBinding.GetBindingRel()
+        token = MaterialCustomControl.strengthTokens[value]
+        UsdShade.MaterialBindingAPI.SetMaterialBindingStrength(directRel, token)
+        # Force update of all children in the VP2 delegate.
+        cmds.evalDeferred(lambda: ufe.Scene.notify(ufe.ObjectRename(item, item.path())))
+
+    def onReplace(self, *args):
+        '''
+        Refresh the UI when the time changes.
+        '''
+        # Nothing needed here since USD data is not time varying. Normally this template
+        # is force rebuilt all the time, except in response to time change from Maya. In
+        # that case we don't need to update our controls since none will change.
+        pass
+
+    def _refreshUI(self):
+        '''
+        Refresh the UI when the data is modified.
+        '''
+        matAPI = UsdShade.MaterialBindingAPI(self.prim)
+        mat, matRel = matAPI.ComputeBoundMaterial()
+        directBinding = matAPI.GetDirectBinding()
+
+        token = 'weakerThanDescendants'
+        if directBinding:
+            directRel = directBinding.GetBindingRel()
+            token = UsdShade.MaterialBindingAPI.GetMaterialBindingStrength(directRel)
+        strength = self.strengthLabels[token]
+
+        if matRel.GetPrim() == self.prim:
+            self._refreshUIForDirect(mat, strength)
+        else:
+            self._refreshUIForInherited(mat, matRel, directBinding.GetMaterialPath(), strength)
+
+    def _refreshUIForDirect(self, mat, strength):
+        '''
+        Refresh the UI when the material is directly on the prim.
+        '''
+        # Note: hide UI elements before filling values.
+        cmds.button(self.gotoPrimButton, command='', e=True)
+        cmds.symbolButton(self.gotoPrimButton, edit=True, enable=False, visible=False)
+        cmds.rowLayout(self.inheritedMatLayout, edit=True, visible=False)
+        cmds.rowLayout(self.fromPrimLayout, edit=True, visible=False)
+
+        self._fillUIValues(mat.GetPath().pathString, '', '', strength)
+
+    def _refreshUIForInherited(self, mat, matRel, directMatPath, strength):
+        '''
+        Refresh the UI when the material is inherited from an ancestor prim.
+        '''
+        direct = directMatPath.pathString if directMatPath else ''
+        inherited = mat.GetPath().pathString
+        fromPath = matRel.GetPrim().GetPath().pathString
+
+        ufePath = ufe.Path([
+            self.item.path().segments[0],
+            ufe.PathSegment(fromPath, mayaUsd.ufe.getUsdRunTimeId(), '/')])
+        ufePathStr = ufe.PathString.string(ufePath)
+        melCommand = 'updateAE "%s"' % ufePathStr
```

Comment:
Another important change is using `updateAE` here instead of `cmds.select`. Thanks!